src/v2/models/CreatePartnerCrossRampTransactionV2RequestBody.ts

```diff
@@ -0,0 +1,41 @@
+import { fiatCurrency } from './Const';
+
+export type CreatePartnerCrossRampTransactionV2RequestBody = {
```

Comment:
You're missing the quoteId, now you can use a quote to have  in ramps operations to get a fixed pre-calculated quote